internal/sync/sync.go

```diff
@@ -129,3 +127,46 @@ func NewCommand(finders ...depbot.FinderFn) *Command {
 		finders: finders,
 	}
 }
+
+func urlOrDefaultFromArgs(args []string) string {
+	url := valueFromArgs(args, "--server-address")
+
+	if url == "" {
+		url = os.Getenv(DepbotServerAddr)
+	}
+	if url == "" {
+		url = "http://app.depbot.com/api/sync"
+	}
+
+	if !strings.Contains(url, "http") {
```

Comment:
This needs to be `strings.HasPrefix` or URL or parsing the url.URL to check if it has a scheme.